doc/_porting.md

```diff
@@ -9,11 +11,19 @@ In particular, remove all calls to deprecated methods and replace them appropria
 
 The vast majority of the changes took place in C++.
 
-# Changes in C++
+\section PortingC Changes in C++
 Although the class system was heavily re-worked, it should not make a lot of difference from the user's point of view.
 Relevant changes are listed below.
 
-## Namespace
+\subsection PortingCHeaderonly Header-only
+Pinocchio is now fully header-only. This means you do not have to link to the Pinocchio library when compiling your code.
+On the other hand, you might need to link to additional system libraries.
+For instance, when using URDF, you now need to add
```

Comment:
We do not have to detail here the link against other libraries.
I think we should keep the description as simple as possible.
The link against the dependencies has to be detailed through CMake or Pkg-config usage.
The way you present here is not safe and generic at all.

Reply:
This was not meant at all to be generic and exhaustive. It was just an example based on what I found when porting the examples in doc-v2. If you think the reference to `libboost_system` and `liburdfdom_model` is potentially confusing, I can certainly remove it, but I would leave a generic warning about linking (I assume the Pinocchio library was thinking about it before).